src/components/__tests__/ratings.test.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 Ultimaker B.V.
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+// component
+import Ratings from '../ratings';
+import HighlightIcon from '../icons/highlight_icon'
+import ApprovedIcon from '../icons/approved_icon'
+
+describe('The Ratings component', () => {
+    it('should render 4.3 of 5 highlight icons', () => {
+        let wrapper = shallow(<Ratings max={5} rating={4.3} Icon={HighlightIcon} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should render a third of 10 approved icons', () => {
```

Comment:
nitpicking: 3 out of 10. a third would be 3.3333333333...

Reply:
The point is exactly to test 3.33333 and see it works correctly. We pass `10 / 3`